uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-files-tab.js

```diff
@@ -85,10 +87,24 @@ define([
     };
 
     (function init () {
-      vm.supportedFileTypes = _.keys(sharedSettings.fileUploader.allowedMimeTypes);
+      loadSupportedFileTypes();
```

Comment:
`vm.request.loadAttachments()` has no relation with `supportedFileTypes`. `supportedFileTypes` is used to show the allowed extensions in the UI only.